### ISSUE TITLE
npm WARN deprecated

### ISSUE THREAD

**Issue by gucong3000** [editorconfig/editorconfig-core-js#65]
```bash
$ npm install -g eclint
npm WARN deprecated @types/commander@2.12.2: This is a stub types definition for commander (https://github.com/tj/commander.js). commander provides its own type definitions, so you don't need @types/commander installed!
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
/home/travis/.nvm/versions/node/v9.11.1/bin/eclint -> /home/travis/.nvm/versions/node/v9.11.1/lib/node_modules/eclint/bin/eclint.js
+ eclint@2.6.0
added 286 packages in 13.368s
```

**Comment by gucong3000** [comment]
```bash
$ npm ls @types/commander
eclint@2.7.0 E:\work\eclint
`-- editorconfig@0.15.0
  `-- @types/commander@2.12.2
```

**Comment by josephfrazier** [comment]
Looks like `@types/commander` was removed from this project in 59f3ab29ff4cfc67591135d0ae569189405ba0b9, so we'd need to:

1. Publish a new version of `editorconfig`
2. Upgrade `eclint` to the new `editorconfig`
3. Upgrade `editorconfig` to the new `eclint`
4. Publish a new version of `editorconfig`

**Comment by josephfrazier** [comment]
I don't know anything new (nor do I have any relevant push/publish permissions), just what's above :/

**Comment by jedmao** [comment]
Published v0.15.1.
